torch/_inductor/ir.py

```diff
@@ -3929,21 +3929,12 @@ def num_reads(self):
 
     @cache_on_self
     def is_non_scalar_tensor_num_reads_larger_than_one(self):
-        data = self.data
-        if isinstance(data, Loops):
-            read_writes = ComputedBuffer(
-                name=None,
-                layout=FlexibleLayout(
-                    device=data.get_device(),
-                    dtype=data.get_dtype(),
-                    size=data.get_size(),
-                ),
-                data=data,
-            ).get_read_writes()
-            return sum(read.index != 0 for read in read_writes.reads) > 1
-        else:
-            # Skip the check for non Loops instances
-            return True
+        # Skip the check for non Pointwise instances
+        return (
+            (sum(read.index != 0 for read in self.data.get_reads()) > 1)
+            if isinstance(self.data, Pointwise)
```

Comment:
If you only care about `Pointwise` here, shall we rename the function name to reflect this?

Reply:
Thanks for the suggestions and changed.